test/integration_test/specs/kubevirt-fedora/fedora-vm.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  #  annotations:
+  #    kubevirt.io/latest-observed-api-version: v1
+  #    kubevirt.io/storage-observed-api-version: v1alpha3
+  labels:
+    app: fedora-vm
+    vm.kubevirt.io/template: fedora-server-small
+    vm.kubevirt.io/template.namespace: openshift
+  name: test-vm-csi
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        vm.kubevirt.io/flavor: small
+        vm.kubevirt.io/os: fedora
+        vm.kubevirt.io/workload: server
+      creationTimestamp: null
+      labels:
+        kubevirt.io/domain: fedora-datavolume
```

Comment:
is this label correct? In my running VM, the domain is same as the name of the VM.
```
    template:
      metadata:
        labels:
          kubevirt.io/domain: fedora-communist-toucan
          kubevirt.io/size: small
 ```

Reply:
Removed